lms/djangoapps/teams/tests/test_csv.py

```diff
@@ -330,6 +343,113 @@ def test_user_moved_to_another_team(self):
         self.assert_learner_removed_emitted(team_1.team_id, audit_learner.id)
         self.assert_learner_added_emitted(team_2.team_id, audit_learner.id)
 
+    def test_exceed_max_size(self):
+        # Given a bunch of students enrolled in a course
```

Comment:
would you mind tossing in some event assertions? assert_no_events_emitted() assert_learner_removed_emitted() and  assert_learner_removed_emitted()?

Reply:
Done :+1: